scripts/declarations_diff.sh

```diff
@@ -1,5 +1,44 @@
 #!/usr/bin/env bash
 
+ : <<'BASH_DOC_MODULE'
```

Comment:
Do you know where this is documented? I'm unable to find this in the bash manual.

Reply:
This is a trick that I learned by googling for multiline comments in bash.  The colon `:` is a no-op command and `<<TEXT...TEXT` is how you can pass the command inside `TEXT` to a previous one, similar to a redirection: you redirect random text as an argument to a command that does nothing with it.  Finally, the single quotes around the first `TEXT` avoid some parsing of the intervening text, but I forgot exactly what.

Reply:
Thanks for explaining. In essence, `BASH_DOC_MODULE` is not a magic variable (you could have written `DamianoWritesRandomStuff` instead), has no further effect - and this is basically a multi-line comment?

Reply:
Yes, exactly: any text that does not "randomly appear" in what follows, until you repeat it gets ignored.  I chose `BASH_DOC_MODULE` to communicate what the intent of the following text was.